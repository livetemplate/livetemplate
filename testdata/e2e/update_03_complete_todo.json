{"10":"2","11":"0","13":"100","31":"2023-01-01 10:45:00"}