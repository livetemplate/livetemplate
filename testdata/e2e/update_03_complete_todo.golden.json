{
  "5": "2",
  "6": "0",
  "7": "100%",
  "8": [
    ["u", "k4c8d", {"0": "completed", "4": "âœ“"}]
  ],
  "9": "2023-01-01 10:45:00"
}